split-webapp/split/src/pages/Login.js

```diff
@@ -94,17 +94,12 @@ function Login() {
                         Log In
                       </Button>
                     </NavLink>
+                    <div>
 
-                    <NavLink to="/SignUp">
-                      <Button
-                        variant="contained"
-                        color="primary"
-                        borderRadius={30}
-                        className={classes.margin}
-                      >
-                        Sign up
-                      </Button>
+                    <NavLink to="/SignUp" className="navLink">
```

Comment:
nit: the classname should probably be something like "SignUpLink" instead of navLink which is a bit general